spikesorters/sorterlist.py

```diff
@@ -93,6 +93,14 @@ def installed_sorters():
     '''
     return sorted(list([s.sorter_name for s in installed_sorter_list]))
 
+def print_sorter_version():
```

Comment:
maybe `print_sorter_versions`? more than one sorter

Reply:
If you want.